config/esp32/components/chip/Kconfig

```diff
@@ -108,6 +108,14 @@ menu "CHIP Core"
             help
                 Build CHIP test binaries.
 
+        config DISPATCH_EVENT_LONG_DISPATCH_TIME_WARNING_THRESHOLD_MS
+            int  "Set threshold in ms"
+            default 700
+            help
+                 Time threshold for events dispatching. By default set to 0 to
```

Comment:
This comment is not correct. Also why are you setting the default to 700ms, vs the default that is used right now of 100?